telegramBot.js

```diff
@@ -0,0 +1,200 @@
+                    console.log(result);
+                    return result;
+                }
+            
+                const fileURL = "http://i.imgur.com/5rOhtdL.png"
+                this.sendFile(fileURL)
+            } else {
+                ctx.reply('Bot uninitialized. Run /init command to initialize.')
+            }
+        })
+
+        this.bot.command('fs_r', async (ctx) =>{
+            if(this.initialzied)  {
+                if(this.onCommandReceived){
+                    const input = ctx.message.text.slice(1).trim().split(' ');
+                    const command = input.shift();
+                    const commandArgs = input.join(' ');
+                    const result = await this.onCommandReceived(command, commandArgs, 'telegram');
+                    console.log(result);
+                    return result;
+                }
+            
+                const fileURL = 'https://homepages.cae.wisc.edu/~ece533/images/airplane.png'
+                this.sendFileRemote(fileURL)
+            } else {
+                ctx.reply('Bot uninitialized. Run /init command to initialize.')
+            }
+        })
+
+        this.bot.use((ctx) => {
+            this.chatID = ctx.chat.id
+            this.chatTitle = ctx.chat.title
+        })
+
+    }
+
+    setCommandListener(commandListener) {
+        this.onCommandReceived = commandListener;
+    }
+
+    async getUpdates(){
+
+        try{
+            const update = await axios.get(this.updateURL);
+            return update.data;  
+        } catch(err){
+            console.log(err)
+        }
+        
+    }
+
+    getChannels(){
+        return {
+            id: this.chatID,
+            name: this.chatTitle
+        }
+    }
+
+    async sendMessageToAllChannels(message){
+        if(this.initialzied){
+            const response = await axios.post( `${this.baseURL}sendMessage?chat_id=${this.chatID}&text=${encodeURIComponent(message)}` );
+            if(response.status = 200){
+                console.log('Message sent successfully')
+            }
+        } else {
+            console.log('Chat Uninitialized. Run /init command in Telegram Client and try again.')
+        }
+    }
+
+    async sendFileLocal(filePath){
+        
+        console.log(`${this.baseURL}sendPhoto`)
+
+        if(this.initialzied){
+
+            var photoURL = fileURL;
+            var formData = {
+                chat_id: this.chatID,
+                photo: await request(photoURL)
+            };
+
+            console.log(photo)
+
+            const response = await axios.post({
+                url: JSON.stringify(`${this.fileURL}sendPhoto`),
+                formData: formData
+            });
+
+            console.log(response)
+        } else {
+            console.log('Chat Uninitialized')
+        }
+    }
+
+    async sendFileRemote(fileURL){
+        if(this.initialzied){
+            try{
+                return this.bot.telegram.sendDocument(this.chatID, fileURL);
```

Comment:
we could use `return this.bot.telegram.sendDocument(this.chatID, fileURL)` directly, no need to do try catch, we can also remove `async` if we not using `await`